src/StorageTest.tsx

```diff
@@ -32,16 +31,8 @@ export async function storageTest() {
   try {
     await set('idb-test', true);
   } catch (e) {
-    errorLog('storage', 'Failed IndexedDB Test - trying to delete database', e);
-    try {
-      await deleteDatabase();
-      await set('idb-test', true);
-      // Report to sentry, I want to know if this ever works
-      reportException('deleting database fixed IDB', e);
```

Comment:
I couldn't find any evidence in Sentry that this had ever been reported, so I removed the deletion behavior.